content/en/guides/core/artifacts/track-external-files.md

```diff
@@ -101,7 +101,7 @@ Based on your use case, read the instructions to enable object versioning: [AWS]
 
 ### Add and download an external reference example
 
-The following code snippet shows how to upload a dataset to an Amazon S3 bucket, track it with a reference artifact, and how to download the artifact:
+The following code sample uploads a dataset to an Amazon S3 bucket, tracks it with a reference artifact, then downloads it:
```

Comment:
Sometimes we say "Amazon S3" and sometimes we just say "S3". Also, you could be using a MiniIO or other S3-compatible bucket.

Reply:
The correct product name is Amazon S3. This is what we should be using everywhere.